documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -1364,10 +1364,29 @@ This property replaces the `mongodb.hosts` property that was available in previo
  +
 [NOTE]
 ====
-Connectors that capture changes from a sharded MongoDB cluster use this connection string only during the initial shard discovery process. 
+Connectors that capture changes from a sharded MongoDB cluster use this connection string only during the initial shard discovery process when xref:mmongodb-property-mongodb-connection-mode[`mongodb.connection.mode`] is set to `replica_set`.
 After the initial discovery process, connection strings are generated for each individual shard.
 ====
 
+|[[mongodb-property-mongodb-connection-mode]]<<mongodb-property-mongodb-connection-mode, `+mongodb.connection.mode+`>>
+|`replica_set`
+|Specifies the strategy used by the connector when connecting to `sharded` MongoDB cluster.
+Set this property to one of the following values:
+
+`replica_set`:: individually connect to each replica set / shard .
```

Comment:
@roldanbob the `or` is a mistake here. Sharded cluster is composed of shard and each shard is a replica set.

Reply:
I updated the description based on that understanding. Let me know if the new version works.

Reply:
This wording is correct.


> @jcechace So, looking at the diagram [here](https://www.mongodb.com/docs/manual/core/sharded-cluster-components/#production-configuration), would the connector attach to the primary node in each shard's replica set?

This is tricky... the exact node the client is communicating with can wary based on configuration (e.g. ReadPreference). For example the client will route write operations always to the primary but can route read operations to one of the secondaries if available (this is actually the configuration used by the connector now). So it's better to think about it as connecting to the Replica Set as a whole rather than connecting to any specific node within it.